pkg/transaction-builder/enums.go

```diff
@@ -0,0 +1,11 @@
+package transactionbuilder
+
+// CoinDenom enum that represents all coin denominations of Pocket
+type CoinDenom string
+
+const (
+	// Upokt represents upokt denomination
+	Upokt CoinDenom = "upokt"
+	// Pokt represents pokt denomination
+	Pokt CoinDenom = "pokt"
+)
```

Comment:
Was wondering: wdyt about adding a `ChainID` enum here as well? It's pretty extra but would be handy as it's not really documented anywhere what `ChainID` corresponds to.

Reply:
I was told by @rem1niscence that these values are not static over time, if they don't I find an enum pretty useful too

Reply:
What do you mean they're not static over time?

Reply:
`ChainID` is set at the genesis file, and could be any string (see [here](https://github.com/pokt-network/pocket-network-genesis/blob/master/mainnet/genesis.json#L3)). The enum could be useful to give a general sense of what is expected, assuming people don't go straight to docs.

Reply:
As long as you don't lock it to the enum, it's fine. You should still be allowed to use any string, but 95% people are using the enum (either connecting to mainnet, testnet or a localnet)